alcs-frontend/src/app/services/notification/notification.service.ts

```diff
@@ -0,0 +1,24 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { firstValueFrom } from 'rxjs';
+import { environment } from '../../../environments/environment';
+import { NotificationDto } from './notification.dto';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class NotificationService {
+  constructor(private http: HttpClient) {}
+
+  async fetchMyNotifications() {
+    return await firstValueFrom(this.http.get<NotificationDto[]>(`${environment.apiRoot}/notification`));
```

Comment:
I think you should be able to remove async and await from function and use simple return

Reply:
Done